scripts/test-sim-benchmark.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ueo pipefail
+
+# Purpose: This script executes uplink upload and download benchmark tests against storj-sim.
+# Setup: Remove any existing uplink configs.
+# Usage: from root of storj repo, run
+#   $ storj-sim network test bash ./scripts/test-sim-benchmark.sh
+# To run and filter out storj-sim logs, run:
+#   $ storj-sim -x network test bash ./scripts/test-sim-benchmark.sh | grep -i "test.out"
+
+SATELLITE_0_ADDR=${SATELLITE_0_ADDR:-127.0.0.1}
+
+apiKey=$(storj-sim network env GATEWAY_0_API_KEY)
```

Comment:
this wont work on Linux. here we need `export apiKey=$(storj-sim network env GATEWAY_0_API_KEY)`